dracut.sh

```diff
@@ -1949,24 +1949,36 @@ if [[ $no_kernel != yes ]]; then
 
     if [[ $drivers ]]; then
         # shellcheck disable=SC2086
-        hostonly='' instmods $drivers
+        if ! hostonly='' instmods $drivers; then
```

Comment:
Without the `-c` option, `instmods` always returns 0, so this check is not necessary.

Reply:
This is correct. I missed it and will fix, no check of exit status needed here.

but in below 3 places, it can certainly return non zero value and error goes unnoticed.